pkg/deploy/strategy/support/lifecycle.go

```diff
@@ -397,6 +397,7 @@ func makeHookPod(hook *deployapi.LifecycleHook, rc *kapi.ReplicationController,
 					Env:             mergedEnv,
 					Resources:       resources,
 					VolumeMounts:    volumeMounts,
+					SecurityContext: baseContainer.SecurityContext,
```

Comment:
yes and should not be copied from baseContainer but from the deployment config...